WORKSPACE

```diff
@@ -47,7 +47,7 @@ go_rules_dependencies()
 
 go_download_sdk(
     name = "go_sdk",
-    version = "1.19",
```

Comment:
Doesn't setting this to 1.19 mean that the latest 1.19 patch version of go_sdk is downloaded? Do we need this change?

Reply:
never mind this implies 1.19.0 and not latest patch version of 1.19.